Add -u option to filter out "(used in module)" results (#202)
nadeesha/ts-prune: README.md

- `-i, --ignore` - errors ignore RegExp pattern
- `-e, --error` - return error code if unused exports are found
- `-s, --skip` - skip these files when determining whether code is used. (For example, `.test.ts?` will stop ts-prune from considering an export in test file usages)

> - `-u, --unusedInModule` - do not display "(used in module)" unused exports

Reviewer: ```suggestion
- `-u, --unusedInModule` - Skipped files that are used in module (marked as `used in module`)
```

Also , I would suggest to use the same description in the README and then configurator.ts for consistency
